libweb3jsonrpc/Eth.cpp

```diff
@@ -275,43 +275,14 @@ string Eth::eth_sendTransaction(Json::Value const& _json)
 			throw JsonRpcException("Unknown authentication error.");
 		}
 	}
-	catch (ZeroSignatureTransaction&)
+	catch (JsonRpcException&)
```

Comment:
Now to avoid this weird pattern surfacing up again, can we maybe `throw / catch(Exception const&)` around just `submitTransaction` call?

Or alternatively - instead of `throw JsonRpcException` in `case`s above assign some variable like `string authenticateError` and then do `throw JsonRpcException(authenticateError)` after `catch Exception`.